README.md

```diff
@@ -66,6 +66,20 @@ katex.render("c = \\pm\\sqrt{a^2 + b^2}\\in\\RR", element, {
 
 Math on the page can be automatically rendered using the auto-render extension. See [the Auto-render README](contrib/auto-render/README.md) for more information.
 
+### Font size and lengths
+
+KaTeX uses the CSS <em>em</em> unit to render all lengths, so everything
+scales according to the CSS <code>font-size</code> that you use
+on the output KaTeX element.
+Internally, KaTeX imagines that the base font size (1em) is 10pt (the default
+in LaTeX), and all LaTeX lengths are defined relative to that assumption.
+For example, <code>\arraycolsep</code> defaults to 5pt in LaTeX,
+which translates to 0.5em in CSS (because 1em = 10pt).
+Similarly, if you specify a length using LaTeX absolute units,
+such as <code>\rule{1cm}{1pt}</code>, it gets converted into the
+equivalent number of ems assuming a 10pt font, which will end up getting
+scaled according to your font size.
```

Comment:
I wondered if this documentation needed updating -- thanks for the comments.  I wasn't aware of the 1.21x scaling.  I assume there's a reason for that, like KaTeX fonts integrate better with typical web fonts at that scale?

Anyway, I took a stab at rewriting, and justifying why this makes sense.  Namely, "The end result is that any rendered KaTeX should be a scaled version of what LaTeX would do with a 10pt base font (e.g., `\documentclass{article}`), where the scale depends on the CSS `font-size`."  This is the *only* definition that will satisfy this property.  Personally, I think it's what I'd want -- I want to be able to copy/paste some KaTeX code into LaTeX (or vice versa), even code that uses absolute units, and have everything look relatively the same, just up to a global scale factor.

To me, mapping KaTeX `pt` (say) to CSS `pt` would not make sense, precisely because an author wouldn't have enough context when writing the LaTeX to know how that should be scaled to "look good".  For example, maybe you make an equation look good, but then globally change the site's font-size.  Now your formulas will look relatively bad.

On the other hand, I'd be fine with later adding a unit like `css pt` that maps to CSS pt.  Except that, as you say, this would make height computation difficult if not impossible to do on the server side.